plugin-template

```diff
@@ -331,7 +332,11 @@ def main():
         print("\nAn updated plugin template config written to {path}.\n".format(path=file_path))
 
     if "github" in sections:
-        migrate_pytest_pin(plugin_root_dir)
+        utils.migrate_changelog_to_markdown(
+            plugin_root_dir,
+            gh_org=config.get("github_org", "pulp"),
```

Comment:
```suggestion
            gh_org=config["github_org"],
```
The defaults must have been applied at this point. Also repeating the same (default) information is a bad idea.